support-frontend/test/utils/CheckoutValidationRulesTest.scala

```diff
@@ -182,7 +187,7 @@ object TestData {
     lastName = "hopper",
     product = DigitalPack(Currency.USD, Monthly),
     firstDeliveryDate = None,
-    paymentFields = StripePaymentFields("test-token"),
+    paymentFields = StripePaymentMethodPaymentFields("test-token"),
```

Comment:
I haven't looked in detail at the tests that use these, but are we missing anything by not having a `StripeSourcePaymentFields` request now?

Reply:
I suppose it is temporary anyway

Reply:
We do have one, that's on line 108 actually. Due to the way these tests work it start with the default and overwrites it